cdpctl/cli.py

```diff
@@ -93,9 +94,16 @@ def skeleton(output_file) -> None:
     render_skeleton(output_file=output_file)
 
 
+@click.command()
```

Comment:
can we make it possible that it works with all combinations of `-v`, `-V` or `--version`

Reply:
how about I just make it -v and --version?